content/components/core/code/scss-variables.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: section
+---
+
+## SCSS Variables
+
+### Default Theme Colours
+
+| Variable Name               | Default Value | Description                                                                                                |
+|-----------------------------|---------------|------------------------------------------------------------------------------------------------------------|
+| `$AU-color-foreground-text`   | `#313131`       | Colour used for paragraphs, headings etc.                                                                                 |
+| `$AU-color-foreground-action` | `#00698F`       | Colour of links, buttons or other action items.                                                             |
```

Comment:
Not sure I know what `or other action items. ` are, maybe say `buttons and interactive elements`